tree_dialog/treeDialog_view.py

```diff
@@ -9,6 +9,11 @@
 
 
 class TreeDialog(SelectDialog):
+    """Widget for displaying data (represented as list) in tree form
+    aside from tree view, dialog inherits SelectDialog, therefore provides displaying
```

Comment:
Maybe better to say "TreeView" meaning Qt Widget.

Reply:
Comment was meant to relate to form of representation, not connection to the particular widget.